README.md

```diff
@@ -19,10 +20,10 @@
     mkvirtualenv -p python3 just-spaces
     ```
 
-3. Install the requirements.
+3. Install the Django requirements.
 
     ```bash
-    pip install -r requirements.txt
+    pip install -r requirements.txt -r dev-requirements.txt
```

Comment:
Could you tell me a bit about the advantages of having, er, "topical" `requirements.txt` files?